vivisect/symboliks/common.py

```diff
@@ -81,6 +81,7 @@ def __init__(self):
         self.kids          = []
         self.parents       = []
         self.cache         = {}
+        self._reduced       = False
```

Comment:
The only problem here is "incremental" calls to reduce...  if we were to reduce and then continue adding nodes ( way beneath this one) the reduce cache would be incorrect.  If we want this optimization ( which i think we do ) we need to find a way to account for that....